pkg/util/util.go

```diff
@@ -1023,3 +1040,15 @@ func WriteYamlFile(name string, obj runtime.Object, moreObjects ...runtime.Objec
 
 	return nil
 }
+
+// GetEnvVariable is looknig for env variable called name in env and return a pointer to the variable
+func GetEnvVariable(env *[]corev1.EnvVar, name string) *corev1.EnvVar {
+	for i := range *env {
+		e := (*env)[i]
+		if e.Name == name {
+			return &e
+		}
+	}
+	log.Printf("❌ Not Found Env variable: %s\n", name)
```

Comment:
would remove the log here. sometimes it is expected to not find an env and the logs are just annoying. let the caller decide if it should be logged